### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by DariuszPorowski**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -55,8 +55,8 @@ echo "INPUT_LOG_LEVEL: ${INPUT_LOG_LEVEL}"
 echo "----------------------------------"
 
 echo "Setting Git safe directory (CVE-2022-24765)"
-echo "git config --global --add safe.directory ${INPUT_SOURCE}"
-git config --global --add safe.directory "${INPUT_SOURCE}"
+echo "git config --global --add safe.directory ${GITHUB_WORKSPACE}"
```
Comment (lawndoc) [inline entrypoint.sh:58]: is this a GitHub environment variable? If so, does it need to follow the double bracket syntax? i.e. `${{ GITHUB_WORKSPACE }}`

File: entrypoint.sh [hunk 2]
```diff
@@ -98,11 +98,11 @@ fi
...
-echo "::set-output name=result::${GITLEAKS_RESULT}"
-echo "::set-output name=command::${command}"
-echo "::set-output name=exitcode::${exitcode}"
+echo "output=${OUTPUT}" >>$GITHUB_OUTPUT
+echo "report=gitleaks-report.${INPUT_REPORT_FORMAT}" >>$GITHUB_OUTPUT
+echo "result=${GITLEAKS_RESULT}" >>$GITHUB_OUTPUT
+echo "command=${command}" >>$GITHUB_OUTPUT
+echo "exitcode=${exitcode}" >>$GITHUB_OUTPUT
 
 echo "Gitleaks Summary: ${GITLEAKS_RESULT}" >>$GITHUB_STEP_SUMMARY
```
Comment (lawndoc) [inline entrypoint.sh:107]: same comment here, I think the above variable would now be saved in `${{ env.GITLEAKS_RESULT }}` after the changes

File: entrypoint.sh (hunk 1 above)
Comment (DariuszPorowski) [reply entrypoint.sh:58]: ${{ ... }} works on GitHub Workflow yaml files only

File: entrypoint.sh (hunk 2 above)
Comment (DariuszPorowski) [reply entrypoint.sh:107]: its built in GH feature: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary

**Pull request merged**